src/test/java/com/exasol/adapter/dialects/oracle/OracleScalarFunctionsIT.java

```diff
@@ -153,4 +147,14 @@ public Connection createExasolConnection() throws SQLException {
             return SETUP.getExasolContainer().createConnection();
         }
     }
+
+    @Override
+    protected void beforeAllSetup() throws SQLException {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    }
+
+    @Override
+    protected void afterAllTeardown() throws SQLException {
+
+    }
```

Comment:
```suggestion
```
Is it possible to remove this when it's empty? If this is required please add a short comment with an explanation

Reply:
No, it is an abstract protected method

Reply:
I added comments